src/Core/src/Platform/Android/FocusExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Android.Runtime;
+using Android.Views;
+using AView = Android.Views.View;
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	public static class FocusExtensions
+	{
+		public static AView? FocusSearch(this AView focused, [GeneratedEnum] FocusSearchDirection direction)
```

Comment:
This implementation is simple, we use `FocusFinder` to move the focus to the next element. In Xamarin.Forms we had https://github.com/xamarin/Xamarin.Forms/blob/caab66bcf9614aca0c0805d560a34e176d196e17/Xamarin.Forms.Platform.iOS/VisualElementRenderer.cs#L194

Where we looked for the element with the next index using the `ITabStopElement `property. But, now we do not have this property. Would this be correct, or can we do something better?

Reply:
We woud need to check what the screen reader behavior is - does the TalkBack focus move with the keyboard focus? If it's inconsistent, we need to figure out what the desired behavior is there. If it's consistent, then I think this is an OK solution

Reply:
We shouldn't copy this code from XF. 100 percent of the code in XF that influenced focus is wrong and was most likely created to work around fundamental flaws of things that XF made clickable and focusable that stacks of hacks had to be built against.

Reply:
sorry, I misread which APIs were being used - taking back what I said and seconding Shane here!

Reply:
If we want to enable this behavior then we should do it through other means like a behavior "https://github.com/CommunityToolkit/Maui/issues/36"

Or through additional APIs so it's opt in